app/controllers/baskets_controller.rb

```diff
@@ -9,6 +9,6 @@ def destroy
     @basket = current_basket
     @basket.destroy
     session[:basket_id] = nil
-    redirect_to shop_url, notice: t('baskets.destroy.notice')
+    redirect_to basket_url, notice: t('baskets.destroy.notice')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.